src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search-form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.icon_small {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.search-input {
+  font-family: inherit;
+  width: 100%;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.search-input_big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.search-input_big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-input_small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.search-input_small::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.search-input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
```

Comment:
you don't need this on `focus/hover`

Reply:
Is only "border: 1px solid #e1e7ed;" redundant?